lib/react_client/utils.dart

```diff
@@ -0,0 +1,74 @@
+library react_client.utils;
+
+import 'package:react/react_client/js_backed_map.dart';
+import 'package:react/react_client/react_interop.dart';
+
+import 'package:react/src/react_client/event_prop_key_to_event_factory.dart';
+import 'package:react/src/react_client/event_factory.dart';
+
+/// Prepares [children] to be passed to the ReactJS [React.createElement] and
+/// the Dart [react.Component].
+///
+/// Currently only involves converting a top-level non-[List] [Iterable] to
+/// a non-growable [List], but this may be updated in the future to support
+/// advanced nesting and other kinds of children.
+dynamic listifyChildren(dynamic children) {
```

Comment:
Hm, since these primarily used by `component_factory.dart`, is there a reason these functions aren't included in that file instead of making new public entrypoint?

Reply:
Nope, I can group them together again. I thought it was nice to keep the little utilities in their own home to keep files like `component_factory.dart` as focused as possible, but I can combine things!